processor/k8sattributesprocessor/README.md

```diff
@@ -26,16 +26,39 @@ The processor stores the list of running pods and the associated metadata. When
 to the pod from where the datapoint originated, so we can add the relevant pod metadata to the datapoint. By default, it associates the incoming connection IP
 to the Pod IP. But for cases where this approach doesn't work (sending through a proxy, etc.), a custom association rule can be specified.
 
-Each association is specified as a list of sources of associations. A source is a rule that matches metadata from the datapoint to pod metadata.
+Each association is specified as a list of sources of associations. The maximum number of sources within an association is 4. 
+A source is a rule that matches metadata from the datapoint to pod metadata.
 In order to get an association applied, all the sources specified need to match.
 
 Each sources rule is specified as a pair of `from` (representing the rule type) and `name` (representing the attribute name if `from` is set to `resource_attribute`).
 The following rule types are available:
 
   - `connection`: Takes the IP attribute from connection context (if available). In this case the processor must appear before any batching or tail sampling, which remove this information.
   - `resource_attribute`: Allows specifying the attribute name to lookup in the list of attributes of the received Resource. Semantic convention should be used for naming.
-
-Pod association configuration.
+   Available attribute names for association sources are the following:
+    - k8s.namespace.name
+    - k8s.pod.name
+    - k8s.pod.hostname
+    - k8s.pod.ip
+    - k8s.pod.start_time
+    - k8s.pod.uid
+    - k8s.replicaset.uid
```

Comment:
Shouldn't these be either enabled by default or explicitly set by the `metadata` rules in order to be available in the association? If so, I think it makes sense to mention that.

Reply:
good point, I have moved the list to the section about extending the metadata and added a remark that they need to be set there to be available in the association